### ISSUE TITLE
Help! Where does glastopf store comments?

### ISSUE THREAD

**Issue by monsterhunterboy98** [mushorg/glastopf#295]
I don't seem to get an login error message after entering random username & passwords. The page simply refreshes?

![image](https://user-images.githubusercontent.com/45062471/48894575-06ab9c00-ee7e-11e8-811b-a0992c0909cd.png)

I am currently exploring around the glastopf direcotries as i just installed it... May I know where does glastopf store all the incoming "blog comments". If I'm not mistaken, once a comment go entered through a form it is normally inserted into a mysql database? My apologies for my lack of understanding I am new to linux 

![image](https://user-images.githubusercontent.com/45062471/48894620-2216a700-ee7e-11e8-8f71-f3a214990e55.png)

**Comment by monsterhunterboy98** [comment]
> The login for is not supposed to show an error as we want it seem like you logged it (returning a 200 status code). The comment is probably in request_raw.
> Glastopf is not supported anymore, please use https://github.com/mushorg/snare

Thank you for your reply! Is glastopf able to emulate a successfully SQL injection attack by injecting it at the login inputs on the page?

**Comment by glaslos** [comment]
yes, but it depends on the type of SQL injection. SNARE has much better SQLi support

**Comment by monsterhunterboy98** [comment]
> yes, but it depends on the type of SQL injection. SNARE has much better SQLi support

-Thank you so much for your reply. I attempted to inject "%' OR '1'='1'" at the Login field and the page simply refreshes. Until now I am still unable to emulate an successful sql injection attack and the login fields.

-I ran sqlmap and attempted to inject SQL queries at the URL and I keep on getting "Internal Server Error" are you able to help me on this? I attempted the same thing on DVWA it respond accordingly to what I injected but Glastopf output the follwing: 
![image](https://user-images.githubusercontent.com/45062471/49004297-ef7eee00-f19e-11e8-8a7f-9b366d536af2.png)

SQLmap:
![image](https://user-images.githubusercontent.com/45062471/49004426-340a8980-f19f-11e8-86d5-25a838e6dba4.png)

Do you think I should just use snare and stop using glastopf?

**Comment by glaslos** [comment]
Yes, please use SNARE, I don't have the time to support Glastopf anymore and to be honest there is no reason considering we have SNARE :)
